src/main/java/org/shanerx/tradeshop/TradeShop.java

```diff
@@ -183,205 +144,9 @@ private void addDirections() {
         ArrayList<BlockFace> allowed = new ArrayList<>();
         allowed.addAll(Arrays.asList(BlockFace.DOWN, BlockFace.WEST, BlockFace.SOUTH, BlockFace.EAST, BlockFace.NORTH, BlockFace.UP));
 
-        for (String str : getSettings().getStringList("allowed-directions")) {
+        for (String str : Setting.ALLOWED_DIRECTIONS.getStringList()) {
```

Comment:
Same here, a java-8 one-liner is definitely fancier than the old-fashioned “enhanced” for-loop.

Reply:
The lambda expression doesn’t have to be one line long, multi-line lambdas are absolutely fine.